public_html/wp-content/mu-plugins/blocks/assets/src/speakers/block-content.js

```diff
@@ -118,18 +150,29 @@ class SpeakersBlockContent extends Component {
 												<a
 													className={ 'wordcamp-speaker-session-link' }
 													href={ session.link }
+													// is there an esc_url() equivalent in g? or do we not need it because we're returning an Element, which gets escaped automatically?
 												>
 													{ decodeEntities( session.title.rendered.trim() ) || __( '(Untitled)', 'wordcamporg' ) }
 												</a>
 												<br />
 												<span className={ 'wordcamp-speaker-session-info' }>
 													{ session.session_track.length &&
+														// will this fatal similar to https://meta.trac.wordpress.org/changeset/8025/ ?
+														// if so, it'd be good to check for other instances of that problem throughout the plugin
+
 														sprintf(
 															/* translators: 1: A date; 2: A time; 3: A location; */
 															__( '%1$s at %2$s in %3$s', 'wordcamporg' ),
 															session.session_date_time.date,
 															session.session_date_time.time,
 															get( find( tracks, ( value ) => { return parseInt( value.id ) === head( session.session_track ) } ), 'name' )
+															// this is pretty difficult for a human to parse when it's all written as a single line, can it be broken up into more digestable chunks?
+															// i don't think it needs to be a named function, but just spreading it across multiple lines would increase the readability a lot
+
```

Comment:
> can it be broken up into more digestable chunks?

Worth trying. I'm not sure why I left it in one line, because I usually like lots of whitespace :D

It might have been an esLint issue.

Reply:
Ah, that'd make sense. Feel free to modify `.eslintrc.js` as you see fit, it's definitely a rough draft. 

My take is that if a rule will create false positives more than ~5% of the time, it's worth turning it off and just relying on our judgement to stick to the standard. Too many false positives and we won't be able to trust the tool, and it'll become more of a nuisance than a helper.

If we ever run into a situation where one of us commits a rule change that others disagree with, we can start a discussion and come to a consensus. I don't think that'll happen too often, though, so better to just go ahead and commit what you think is best, rather than getting bogged down in discussion first.

cc @vedanshujain, same goes for you for modifying the rules.